Tests/Command/DumpCommandTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Bazinga\JsTranslationBundle\Tests\Command;
+
+use Bazinga\Bundle\JsTranslationBundle\Command\DumpCommand;
+use Bazinga\Bundle\JsTranslationBundle\Tests\WebTestCase;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * @covers \Bazinga\Bundle\JsTranslationBundle\Command\DumpCommand
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class DumpCommandTest extends WebTestCase
+{
+    private $transDumper;
+
+    public function setUp()
```

Comment:
should be protected, not public

Reply:
Yup, I was just going with the flow as the one in the local `WebTestCase` was public. I've changed both.